Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Repository/NodeFactory.php

```diff
@@ -77,17 +79,14 @@ public function mapNodeRowToNode(
             : null;
 
         return Node::create(
-            ContentSubgraphIdentity::create(
-                $this->contentRepositoryId,
-                $contentStreamId ?: ContentStreamId::fromString($nodeRow['contentstreamid']),
-                $dimensionSpacePoint ?: DimensionSpacePoint::fromJsonString($nodeRow['dimensionspacepoint']),
-                $visibilityConstraints
-            ),
+            $this->contentRepositoryId,
+            // todo use actual workspace name
+            WorkspaceName::fromString('missing'),
```

Comment:
Doesn't this need to happen in this change?

Reply:
nope its the Postgres adapter ;)